app/lib/pages/send_page.dart

```diff
@@ -33,6 +37,14 @@ class _SendPageState extends State<SendPage> with Refena {
   Device? _myDevice;
   Device? _targetDevice;
 
+  @override
+  void dispose() {
+    super.dispose();
+    if (Platform.isWindows) {
```

Comment:
Extract platform-specific handling to utils (here you can do a noop for Linux (?) and macOS)